build-script.py

```diff
@@ -757,6 +853,7 @@ def main():
         # targets simultaneously. For now, just build one product after the other.
         builder.build("SwiftSyntax")
         builder.build("SwiftSyntaxParser")
+        builder.build("SwiftSyntaxBuilder")
```

Comment:
I think we should also build SwiftSyntaxBuilderGeneration here to make sure it’s actually compatible with the current SwiftSyntaxBuilder, just to be sure.